earth2mip/inference_medium_range.py

```diff
@@ -148,9 +148,13 @@ def process(initial_time):
 
             lead_time = valid_time - initial_time
             logger.debug(f"{valid_time}")
-            # TODO make this more performant grabs all history steps unnecessarily
-            verification_torch = initial_conditions.get_initial_condition_for_model(
-                time_loop=model, data_source=data_source, time=valid_time
+            verification_torch = initial_conditions.get_data_from_source(
+                data_source=data_source,
+                time=valid_time,
+                channel_names=model.out_channel_names,
```

Comment:
should this be `model.in_channel_names`
see #157

Reply:
No. This just gets verification data. The outputs are compared against verification...not the inputs.

Reply:
in graphcast `model.out_channel_names` includes `tp06` that is not in ERA5 so its not clear to me if this would work

Reply:
In practice, graphcast is scored against hdf5 data that have this channel. If the output_channels don't exist in the data that is a bug with the data, not with this routine.